pkg/scheduler/profile/profile.go

```diff
@@ -26,14 +26,15 @@ import (
 	"k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/client-go/tools/events"
 	"k8s.io/kubernetes/pkg/scheduler/apis/config"
+	frameworkruntime "k8s.io/kubernetes/pkg/scheduler/framework/runtime"
```

Comment:
you should only rename to frameworkruntime when there are import collisions.

Please check for this and other places.

Reply:
there is an import collision here, please check 4 lines above: `"k8s.io/apimachinery/pkg/runtime"`